pom.xml

```diff
@@ -41,6 +41,14 @@
                     <includeTestSourceDirectory>true</includeTestSourceDirectory>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>17</source>
+                    <target>17</target>
+                </configuration>
+            </plugin>
```

Comment:
What is the reason to add this?

Reply:
> What is the reason to add this?

I made switch/case for sealed classes. I somehow changed project settings and it wasn`t pass tests on github.